src/reload.lisp

```diff
@@ -45,8 +45,17 @@
                 "/sw/lib/libcrypto.dylib"        ;; Fink
                 "/usr/local/opt/openssl/lib/libcrypto.dylib" ;; Homebrew
                 "/usr/local/lib/libcrypto.dylib" ;; personalized install
-                "libcrypto.dylib"                ;; default system libcrypto, which may have insufficient crypto
-                "/usr/lib/libcrypto.dylib"))
+                ;; System-provided libraries. These must be loaded with an explicit
+                ;; API version, or else the process will crash.
+                ;;
+                ;; Please note that in macOS >= 11.0, these paths may not exist in the
```

Comment:
Why do you say "these paths"? All of them may not exists, or only the lower versions like 35? Can the version 44 not exist?

Reply:
None of these paths exist in the filesystem on my machine, which is running macOS 11.2.2 at this time. I try to explain it in the comment here because someone might have issues with loading the library later, and then that person would go looking for those files and be surprised that they are not present at all. At least that's what happened to me.

Reply:
You can read more about this in the macOS Big Sur release notes: https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-release-notes

(search for dynamic linker)

> New in macOS Big Sur 11.0.1, the system ships with a built-in dynamic linker cache of all system-provided libraries. As part of this change, copies of dynamic libraries are no longer present on the filesystem. Code that attempts to check for dynamic library presence by looking for a file at a path or enumerating a directory will fail. Instead, check for library presence by attempting to dlopen() the path, which will correctly check for the library in the cache. (62986286)